fireant/tests/slicer/queries/test_builder.py

```diff
@@ -258,10 +258,10 @@ def test_build_query_with_multiple_dimensions_and_visualizations(self):
         query = slicer.data \
             .widget(f.DataTablesJS([slicer.metrics.votes, slicer.metrics.wins])) \
             .widget(f.HighCharts(
-              axes=[
-                  f.HighCharts.PieChart([slicer.metrics.votes]),
-                  f.HighCharts.ColumnChart([slicer.metrics.wins]),
-              ])) \
+            axes=[
```

Comment:
please revert all of these changes to indentation

Reply:
Fixed